README.md

```diff
@@ -12,6 +12,24 @@ $ docker run -d -p 2015:2015 abiosoft/caddy
 
 Point your browser to `http://127.0.0.1:2015`.
 
+> Be aware! If you don't bind mount the location certificates are saved to, you may bypass Lets Encrypt limits
+> rending further certificate generation disallowed! See "Saving Certificates" below!
```

Comment:
"bypass" sounds like it won't be a problem; maybe rephrase, something like this:

> ... you could hit your CA's rate limits, preventing obtaining or renewing certificates for a fixed time. ...